test/integration/fixtures/options/watch/test-global-leak.fixture.js

```diff
@@ -0,0 +1,14 @@
+describe('leak detection', function() {
+  function testMe() {
+    x = 123; // leak variable
+  }
+
+  it('should not leak', function(done) {
+    testMe();
+    done(); // just pass
+  });
+
+  before(function() {});
+
+  after(function() {});
```

Comment:
hooks needed here?

Reply:
because `Runner.prototype.checkGlobals()` is invoked every test-end-event and hook-end-event. (https://github.com/mochajs/mocha/blob/master/lib/runner.js#L136-L141)

Reply:
That makes sense, but isn't intuitive from the test. Request: how about a quick comment explaining why these are here?